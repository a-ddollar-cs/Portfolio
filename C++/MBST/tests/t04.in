37	205	
0	23	2
0	21	5
0	1	6
0	36	18
0	19	20
0	22	21
0	2	25
0	13	29
1	34	7
1	3	12
1	22	32
2	22	1
2	11	14
2	33	30
2	16	39
2	8	136
2	20	184
3	5	4
3	4	8
3	20	9
3	33	10
3	23	16
3	34	23
3	18	36
3	35	38
3	29	42
3	16	67
3	25	195
3	12	231
3	7	235
4	8	13
4	25	15
4	17	17
4	21	24
4	15	26
4	6	28
4	10	33
4	28	108
4	9	163
5	14	11
5	13	132
6	32	76
6	25	86
6	20	87
6	16	98
6	29	99
6	27	177
6	28	185
6	23	214
7	31	56
7	8	68
7	36	77
7	24	88
7	25	121
7	18	133
7	33	134
7	10	164
7	30	170
8	22	3
8	11	22
8	31	89
8	13	158
8	16	165
9	35	27
9	36	69
9	11	90
9	17	116
9	33	196
10	17	31
10	31	53
10	14	91
10	19	137
10	24	155
10	13	166
10	28	202
11	22	19
11	21	40
11	14	73
11	30	109
11	31	117
11	27	197
11	33	203
12	34	37
12	36	61
12	35	78
12	31	92
13	15	35
13	20	79
13	25	122
13	21	123
13	36	151
14	20	34
14	32	100
14	22	102
14	30	110
14	35	178
14	29	179
14	33	222
15	23	111
15	22	127
15	33	159
15	30	215
15	34	236
16	28	43
16	33	57
16	23	62
16	20	103
16	32	180
16	25	204
17	28	47
17	25	80
17	34	138
17	32	181
18	19	44
18	26	48
18	21	63
18	33	81
18	28	125
18	24	175
18	31	243
19	31	58
19	24	82
19	28	93
19	27	152
19	34	167
19	30	171
19	25	182
19	20	223
19	29	224
20	30	118
20	29	148
20	27	156
20	26	186
20	32	198
20	31	211
20	35	225
21	25	83
21	30	94
21	33	128
21	26	140
21	35	192
21	27	244
22	35	49
22	24	54
22	36	112
22	30	113
22	26	160
22	29	232
23	26	95
23	29	96
23	31	104
23	32	105
23	25	161
23	24	216
23	30	226
24	25	74
24	28	84
24	27	114
24	34	141
24	36	146
24	29	217
24	33	237
25	35	59
25	29	64
25	30	129
25	31	168
25	34	238
26	28	106
26	29	130
26	34	207
26	36	218
27	31	70
27	33	172
27	29	193
27	32	219
27	36	227
27	34	228
27	30	239
27	35	245
28	30	50
28	32	187
28	35	240
29	35	119
29	32	173
29	36	189
29	33	208
29	30	233
30	31	65
30	32	142
30	34	153
30	33	199
30	35	241
31	34	71
31	32	143
31	33	190
31	36	212
32	35	51
32	33	205
32	36	209
32	34	229
33	34	144
33	35	200
33	36	220
34	36	149
35	36	45
