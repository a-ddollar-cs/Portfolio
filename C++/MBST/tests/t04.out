27 31 70
